web/src/screens/Dashboard/Sidebar.tsx

```diff
@@ -3,6 +3,12 @@ import { useCompanyStore } from '../../store/useCompanyStore';
 const Sidebar = () => {
     const { positions, selectedPosition, selectPosition } = useCompanyStore();
 
+    if (positions.length === 0) {
```

Comment:
Is this temporary? When companies don't have positions when starting out, will it keep showing 'loading...'?